examples/experimental/Federated Learning with Grid Nodes (After remove sleep).ipynb

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
You can remove this file since it's just a copy of the (Before remove sleep) notebook

Reply:
Done! 19719b4